lib/src/mt_dev.c

```diff
@@ -1594,15 +1594,18 @@ static int dev_if_init_pacing(struct mt_interface* inf) {
 static int dev_if_init_virtio_user(struct mt_interface* inf) {
 #ifndef WINDOWSENV
   enum mtl_port port = inf->port;
+  struct mtl_main_impl* impl = inf->parent;
   uint16_t port_id = inf->port_id;
   int ret;
-  char name[32];
+  char name[IF_NAMESIZE];
   char args[256];
   struct rte_ether_addr addr = {0};
+  int sockfd;
+  struct ifreq ifr;
 
   rte_eth_macaddr_get(port_id, &addr);
 
-  snprintf(name, sizeof(name), "virtio_user%u", port_id);
+  snprintf(name, sizeof(name), "virtio_user%u", (uint8_t)port_id);
```

Comment:
Why it need cast to uint8_t?

Reply:
The compiler warns that the length of "virtio_user%u" may be larger than IF_NAMESIZE(16).

Reply:
Get it. Maybe you can drop a comment here to help reader to understand the cast.